api/src/main/java/org/pmiops/workbench/rdr/RdrMapper.java

```diff
@@ -0,0 +1,118 @@
+package org.pmiops.workbench.rdr;
+
+import java.sql.Timestamp;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.model.SpecificPopulationEnum;
+import org.pmiops.workbench.model.WorkspaceActiveStatus;
+import org.pmiops.workbench.rdr.model.RdrWorkspace;
+import org.pmiops.workbench.rdr.model.RdrWorkspaceDemographic;
+import org.pmiops.workbench.utils.mappers.MapStructConfig;
+
+@Mapper(config = MapStructConfig.class)
+public interface RdrMapper {
+  @Mapping(target = "modifiedTime", source = "lastModifiedTime")
+  @Mapping(target = "intendToStudy", source = "intendedStudy")
+  @Mapping(target = "diseaseFocusedResearchName", source = "diseaseOfFocus")
+  @Mapping(target = "ethicalLegalSocialImplications", source = "ethics")
+  @Mapping(target = "scientificApproaches", source = "scientificApproach")
+  @Mapping(target = "findingsFromStudy", source = "anticipatedFindings")
+  @Mapping(target = "status", source = "workspaceActiveStatusEnum")
+  @Mapping(target = "workspaceUsers", ignore = true)
+  @Mapping(target = "excludeFromPublicDirectory", ignore = true)
+  @Mapping(target = "focusOnUnderrepresentedPopulations", source = "specificPopulationsEnum")
+  @Mapping(target = "workspaceDemographic", source = "specificPopulationsEnum")
+  RdrWorkspace toRdrModel(DbWorkspace employeeDbEntity);
+
+  ZoneOffset offset = OffsetDateTime.now().getOffset();
+
+  default OffsetDateTime toModelOffsetTime(Timestamp dbTime) {
+    return dbTime.toLocalDateTime().atOffset(offset);
+  }
+
+  default RdrWorkspace.StatusEnum toModelStatus(WorkspaceActiveStatus workspaceActiveStatus) {
+    return workspaceActiveStatus == WorkspaceActiveStatus.ACTIVE
+        ? RdrWorkspace.StatusEnum.ACTIVE
+        : RdrWorkspace.StatusEnum.INACTIVE;
+  }
+
+  default boolean toModelFocusOnUnderrepresentedPopulation(
+      Set<SpecificPopulationEnum> dbSpecificPopulationSet) {
+    return dbSpecificPopulationSet != null && dbSpecificPopulationSet.size() > 0;
+  }
+
+  default RdrWorkspaceDemographic toModelWorkspaceDemographic(
+      Set<SpecificPopulationEnum> dbPopulationEnumSet) {
+    RdrWorkspaceDemographic rdrDemographic = new RdrWorkspaceDemographic();
+
+    rdrDemographic.setAccessToCare(
```

Comment:
nit: you can get around all ternary operators if you just set all the flages to unset first, then iterate over the enums and use a mapping function, sort of like the `PropertyMapper` class. Probably too fancy to bother with, but it's a neat trick.

So it'd be like `AccessToCare.setRdrFlag(rdrDemogrphic);` You can just zip down the set and call the `setRdrFlag()` method on each one for the thing and avoid all the branching.